org.eclipse.dartboard/src/org/eclipse/dartboard/launch/LaunchUtil.java

```diff
@@ -23,13 +28,31 @@
 	 *                 from {@link File#separator}.
 	 */
 	public static void launchDartFile(String dartSdk, String dartFile) {
+		Launch launch = new Launch(null, ILaunchManager.RUN_MODE, null);
```

Comment:
In case someone doubt this line. Yes it is valid to create a launch with null config.

Reply:
Ah, interesting. I was stuck for a while on this class, because I couldn't figure out how to get a hold of a launch configuration.

Reply:
As far as I remember the full manual way to get a launch object is:

> ILaunchManager lm = DebugPlugin.getDefault().getLaunchManager();
ILaunchConfigurationType type = lm.getLaunchConfigurationType("org.eclipse.dartboard.launch");
ILaunchConfigurationWorkingCopy wc = type.newInstance(null, "My Config");
ILaunchConfiguration lc = wc.doSave();
ILaunch l = lc.launch(LaunchManager.RUN_MODE, null);